workshop_app/templates/workshop_app/workshop_status.html

```diff
@@ -0,0 +1,190 @@
+{% extends "workshop_app/base.html" %}
+
+{% block title %}
+    Workshop Status
+{% endblock %}
+
+{% block extra-dependencies %}
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
```

Comment:
Download these JS and CSS libraries, instead of using a CDN.

Reply:
As this project will be hosted online, isn't it good to use a CDN instead so that static files can be delivered faster?

Reply:
It is good to have a CDN, but if we have our own CDN. These CDN services sometimes take lot of time to load, which might lead to CSS or JS failure in UI. Also, when developing I might not have the internet so this will not load.

Reply:
Anyways in production, [Django recommends](https://docs.djangoproject.com/en/3.0/howto/static-files/deployment/) serving static files using a separate apache/Nginx server.

Reply:
Prathamesh, we usually prefer the projects to be self contained, that's why we host the files ourselves.

The files will be served from wherever the static url points to, Django allows us to configure that, this is exectly why we need you to use dynamic static URLs in templates, etc https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#static

You do not need to make the change in this PR, but as discussed earlier, it is expected that you make this change all over the project in a separate PR

Reply:
Okay, got it.